# A Systems Biology Approach to Elucidating Functional and Genomic Signatures in Host-Pathogen Interactions of Triple Infections: Tuberculosis, HIV, and SARS-CoV-2

## Abstract
The convergence of HIV-1, Mycobacterium tuberculosis, and SARS-CoV-2 infections presents a
formidable challenge characterized by heightened immune suppression and persistent viral replication.
Understanding the shared molecular mechanisms underlying these co-infections is essential,
providing a foundation for future translational research and therapeutic discovery. We analyzed
differentially expressed genes from HIV-1/TB co-infection datasets across multiple tissues, including
lung, spleen, liver, and whole blood. This was followed by PPI network analysis and single-cell multigene
expression analysis. Gene set enrichment and pathways enrichment analysis were carried out
to reveal the immune regulatory pathways that are altered during HIV-1/TB infection and SARS-CoV2
infection. Gene expression profiling pointed out critical genes, including CXCR4 TAP1 TSC22D 3 and
FGR, associated with immune suppression, antigen processing, and viral attachment. TSC22D3 and
CXCR4 were found to be crucial immune cell regulators in lung tissue using single-cell analysis. Apart
from this, PPI network analysis revealed that TAP1, TAP2, and CXCR4 are essential hub genes, whereas
the intersectional analysis with the SARS-CoV-2 gene datasets highlighted 27 genes associated with
the immune response and viral persistence. Pathway enrichment analyses noted several processes,
such as protein phosphorylation, apoptosis, and immune evasion, pointing out common mechanisms
across these infections. Our findings suggest shared molecular signatures in HIV-1/TB and SARS-CoV-2
infections, with central genes offering potential therapeutic relevance. Future work will validate these
targets through wet lab experiments to better understand their roles in immune regulation and develop
novel therapies to enhance viral clearance in co-infected patients.

## Supplementary Information
Please see the file `Supplementary_Data.xlsx` for detailed results, including enriched pathways, gene expression tables, and interaction networks.
